advanced_alchemy/extensions/litestar/cli.py

```diff
@@ -372,17 +373,18 @@ async def _drop_all(
     "--dir",
     "dump_dir",
     help="Directory to save the JSON files. Defaults to WORKDIR/fixtures",
-    type=ClickPath(path_type=Path), # pyright: ignore[reportCallIssue, reportUntypedFunctionDecorator, reportArgumentType]
+    type=ClickPath(path_type=Path),  # type: ignore[type-var,unused-ignore] # pyright: ignore[reportCallIssue, reportUntypedFunctionDecorator, reportArgumentType]
```

Comment:
Why add a type ignore and mark it as unused?

Reply:
Because it would pass locally but fail in CI.  I tried numerous different combinations.  Given the magnitude of the PR, it's on the "come back afterwards" list.